CMake/Utilities.cmake

```diff
@@ -1,3 +1,35 @@
+# only fetch target repo for add_subdirectory later
+function(fetch_repo lib_name)
+  set(supported_libs
+      kvspic
```

Comment:
There is no CMakeLists equivalent file for this. We can remove it.

Reply:
yup. done.